src/DonationForms/resources/styles/design-settings/image.scss

```diff
@@ -1,117 +1,158 @@
+            .givewp-layouts-header {
+                background: none;
+            }
+
+            .givewp-design-settings-image--background {
+                position: absolute;
+                top: 0;
+                left: 0;
+                display: flex;
+                flex-direction: column;
+                justify-content: center;
+                height: 204px;
                 width: 100%;
-                left: -6.75rem;
+                margin-bottom: 10rem;
+                --tint-color: rgba(0, 0, 0, 0.3); /* Note: Color needs to be converted to RGB values. */
+                --tint: linear-gradient(90deg, var(--tint-color), var(--tint-color));
+                background-color: rgb(30, 140, 190);
+                background-image: var(--tint, linear-gradient(transparent, transparent)), var(--givewp-design-settings-background-image);
+                background-position: 50%;
+                background-size: cover;
+            }
+
+            .givewp-layouts .givewp-layouts-goal {
+                margin-top: 12rem;
             }
         }
+    }
 
-        &.givewp-donation-form {
-            background: transparent;
-            padding-top: 15rem;
+    &__center {
+        .givewp-form-secure-badge + .givewp-layouts-headerImage {
+            margin-bottom: -1.25rem;
         }
 
-        .givewp-layouts-header {
-            position: relative;
-            margin-top: 1rem;
-            border-radius: 0.5rem;
+        .givewp-layouts-headerImage {
+            width: 100%;
+            min-width: 100%;
+            border-radius: .5rem;
+            margin-bottom: .25rem;
         }
 
-        .givewp-layouts-headerTextWrapper {
-            position: absolute;
-            top: -10rem;
+        .givewp-design-settings-image--center {
+            display: block;
+            border-radius: .5rem;
+            min-width: 100%;
+            height: 311px;
 
-            .givewp-layouts-headerTitle {
-                margin-bottom: var(--givewp-spacing-4);
+            img {
+                width: 100%;
+                height: 100%;
+                border-radius: .5rem;
+                object-fit: cover;
             }
         }
 
-        .givewp-layouts-header::before {
-            content: "";
-            position: absolute;
-            top: -15.5rem;
-            min-width: 65rem;
-            width: 100%;
-
-            height: 436px;
-            z-index: -1;
-            --tint-color: rgba(0, 0, 0, 0.3); /* Note: Color needs to be converted to RGB values. */
-            --tint: linear-gradient(90deg, var(--tint-color), var(--tint-color));
-            background-color: rgb(30, 140, 190);
-            background-image: var(--tint, linear-gradient(transparent, transparent)), var(--givewp-design-settings-background-image);
-            background-size: cover;
+        .givewp-layouts {
+            &-headerTitle > h2, &-headerDescription p {
+                color: var(--givewp-grey-900) !important;
```

Comment:
we want to avoid using `!important` in global styles as it can be hard to override down the chain - can we figure out how not to apply that?